docker-compose.yml

```diff
@@ -158,7 +179,10 @@ services:
     volumes:
       - ./config/dashboard.toml:/tmp/dashboard-config.toml
     depends_on:
-      - hyperswitch-web
+      hyperswitch-server:
```

Comment:
I had left out hyperswitch-server as a dependency so people can run the control center with their local `cargo run`